src/NuGet.Clients/NuGet.VisualStudio/Extensibility/IVsPathContextProvider2.cs

```diff
@@ -42,5 +44,21 @@ public interface IVsPathContextProvider2 : IVsPathContextProvider
         /// <code>InvalidOperationException</code> when it fails to create a context and return appropriate error message.
         /// </throws>
         bool TryCreateSolutionContext(string solutionDirectory, out IVsPathContext2 context);
+
+        /// <summary>
+        /// Attempts to create an instance of <see cref="IVsPathContext"/> containing only the user wide and machine wide configurations.
+        /// If a solution is loaded, note that the values in the path context might not be the actual effective values for the solution.
+        /// If a customer has overriden the `globalPackagesFolder` key or cleared the `fallbackPackageFolders`, these values will be incorrect.
+        /// It is important to keep this scenario in mind when working with this path. To predict differences you can call this in combination with <see cref="TryCreateSolutionContext(out IVsPathContext2)"/>.
+        /// </summary>
+        /// <returns>
+        /// <code>True</code> if operation has succeeded and context was created.
+        /// <code>False</code> otherwise.
+        /// </returns>
+        /// <exception cref="InvalidOperationException">When we fail to create a context and return appropriate error message</exception>
```

Comment:
If we throw an exception when we can't create the context, when will `false` ever be returned?  Given this is a `Try` method, we should never throw, only return false, or we should return the context, not a bool, and remove `Try` from the name?

Reply:
We will never throw. 
Deleting it. 
I am trying to keep the same calling pattern as the rest of the APIs.